### PR TITLE
Small fix to use TNTClusterer with Spatial method

### PR DESCRIPTION
**Pull request by myucel-fnal**

### CODE REVIEW

File: RecoDataProducts/inc/BkgCluster.hh [hunk 1]
```diff
@@ -17,13 +17,13 @@ namespace mu2e
    {
        //Default hit count chosen for compuational efficiency
        BkgCluster() {_hits.reserve(16);}
-       BkgCluster(TwoDPoint point, float time) : _point(point), _time(time) {_hits.reserve(16);_cpoints.addPoint(_point,0);}
+       BkgCluster(TwoDPoint point, float time) : _point(point), _time(time) {_hits.reserve(16);_cpoints.addPoint(_point,0);_pos = point.pos3();_pos = point.pos3();}
```
Comment (brownd1978) [inline RecoDataProducts/inc/BkgCluster.hh:20]: _pos is set twice.  It can also be set on initialization.

**Review by brownd1978 — CHANGES_REQUESTED** [review]
small fix needed

File: RecoDataProducts/inc/BkgCluster.hh (hunk 1 above)
Comment (myucel-fnal) [reply RecoDataProducts/inc/BkgCluster.hh:20]: Ooops, didn't see that in time.

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Pull request merged**